realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -1827,6 +1827,7 @@ public void isValid_removedParent() {
         list.removeAll(SUPPORTED_IS_EMPTY_TYPES);
         list.remove(RealmFieldType.UNSUPPORTED_MIXED);
         list.remove(RealmFieldType.UNSUPPORTED_TABLE);
+        list.remove(RealmFieldType.UNSUPPORTED_DATE);
```

Comment:
`RealmFieldType` is supposed to be in sync with a similar enum in core, so I'm a bit reluctant to remove them.